css-three-column-layout/styles.css

```diff
@@ -0,0 +1,50 @@
+html,
+body {
+  margin: 0;
+  color: white;
+  background-color: rgb(27, 150, 217);
+  font-family: Lato, sans-serif;
+}
+
+h4 {
+  margin-bottom: 10px;
+}
+
+p {
+  font-size: 0.72rem;
+  color: rgb(221, 221, 221);
+  line-height: 15px;
+}
+
+.icon {
+  margin-top: 23px;
+  height: 45px;
+  width: 45px;
+}
+
+.icon-wrapper {
+  margin: auto;
+  background-color: white;
+  border-radius: 50%;
+  height: 90px;
+  width: 90px;
+}
+
+.container {
+  margin: 30% auto;
+  max-width: 80%;
+  text-align: center;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  flex-basis: 100%;
+}
+
+.column-one-third {
+  flex-basis: 27%;
+  margin: 15px 25px 25px 25px;
```

Comment:
If you use `padding` instead of `margin`, I bet you can keep the measurement used for this class to be more accurate, such as `33%`